src/assets/templates/typescript/src/steps/user-field-equals.ts

```diff
@@ -4,6 +4,10 @@
 import { BaseStep, Field, StepInterface } from '../core/base-step';
 import { FieldDefinition, RunStepResponse, Step, StepDefinition } from '../proto/cog_pb';
 
+import { baseOperators } from './../client/constants/operators';
+import * as util from '@run-crank/utilities';
```

Comment:
- [ ] Will need to add `@run-crank/utilities` to the `package.json` template too.